Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,176 @@
+ParserName: ASimWebSessionPaloAltoCortexDataLake
+EquivalentBuiltInParser: _ASim_WebSession_PaloAltoCortexDataLake
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let EventLookup=datatable(DeviceAction:string, DvcAction:string,EventResult:string,EventResultDetails:string)
+    [
+        "alert", "Allow", "Success","200", 
+        "allow", "Allow", "Success", "200",
+        "continue", "Allow", "Success", "200",
+        "override", "Allow", "Success", "200",
+        "block-continue", "Allow", "Partial", "200",
+        "block-url", "Deny", "Failure", "503",
+        "block-override", "Deny", "Failure", "302",
+        "override-lockout", "Deny", "Failure","503",
+        "reset client", "Reset Source", "Failure", "503",
+        "reset server", "Reset Destination", "Failure", "503",
+        "reset both", "Reset", "Failure", "503",
+        "deny", "Deny", "Failure", "503",
+        "drop", "Drop", "Failure", "503",
+        "drop ICMP", "Drop ICMP", "Failure", "503"
+    ];
+  let parser = (disabled: bool=false) {
+      CommonSecurityLog
+      | where not(disabled)
+          and DeviceVendor == "Palo Alto Networks" and DeviceProduct == "LF"
+          and DeviceEventClassID == "THREAT" and Activity == "url"
+      | parse-kv AdditionalExtensions as (PanOSDestinationUUID: string, PanOSDestinationLocation: string, PanOSDestinationDeviceMac: string, PanOSSourceUUID: string, PanOSSourceDeviceMac: string, PanOSReferer: string, PanOSIsClienttoServer:string, PanOSSourceDeviceHost: string, PanOSDestinationDeviceHost:string, start: string) with (pair_delimiter=";", kv_delimiter="=")
+      | invoke _ASIM_ResolveDvcFQDN('DeviceName')
+      | invoke _ASIM_ResolveSrcFQDN('PanOSSourceDeviceHost')
+      | invoke _ASIM_ResolveDstFQDN('PanOSDestinationDeviceHost')
+      | lookup EventSeverityLookup on LogSeverity
+      | lookup EventLookup on DeviceAction
+      | extend
+          EventStartTime = todatetime(start),
+          SrcIpAddr = coalesce(DeviceCustomIPv6Address2, SourceIP),
+          DstIpAddr = coalesce(DeviceCustomIPv6Address3, DestinationIP),
+          HttpRequestMethod = toupper(RequestMethod),
+          NetworkProtocol = toupper(Protocol),
+          NetworkSessionId = tostring(FieldDeviceCustomNumber1),
+          SrcDomain = coalesce(SourceNTDomain, SrcDomain),
+          DstDomain = coalesce(DestinationNTDomain, DstDomain)
+      | project-rename
+          EventProduct = DeviceProduct,
+          EventVendor = DeviceVendor,
+          DvcIpAddr = Computer,
```

Comment:
we mapped Dvc fields with 'DeviceName'
Can you confirm 'Computer' and 'DeviceName' are same devices?